server.py

```diff
@@ -34,7 +32,7 @@ def render_GET(self, request):
         delay = max(0, random.gauss(mu, sigma=mu / 2))
         call = reactor.callLater(delay, self._delayedRender, request)
         request.notifyFinish().addErrback(self._responseFailed, call)
-        print delay
+        print(delay)
```

Comment:
Would be nice to add a ``from __future__ import print_statement``, this won't have an effect with a scalar delay, but if someone changes this to ``print('delay', delay)`` then this won't work as expected in python 2.